src/openMVG/cameras/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ set_property(TARGET openMVG_camera APPEND PROPERTY
 
 target_compile_features(openMVG_camera INTERFACE ${CXX11_FEATURES})
 target_link_libraries(openMVG_camera INTERFACE openMVG_numeric ${cereal_TARGET} ${OPENMVG_LIBRARY_DEPENDENCIES})
-install(TARGETS openMVG_camera DESTINATION lib EXPORT openMVG-targets)
+install(TARGETS openMVG_camera EXPORT openMVG-targets)
```

Comment:
Just to confirm, you see no need of lib since it is a headeronly library?

Reply:
Correct.